### PR TITLE
Add Tech section to README + support static storybook

### PR DESCRIPTION
**Pull request by liatgo**
<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```sh
  npm install @kenshooui/react-tree@0.0.25-canary.161.454.0
  npm install @kenshooui/material-tree@0.0.8-canary.161.454.0
  # or 
  yarn add @kenshooui/react-tree@0.0.25-canary.161.454.0
  yarn add @kenshooui/material-tree@0.0.8-canary.161.454.0
  ```
</details>

### CODE REVIEW

File: packages/core/README.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ Examples can be found - [here](https://github.com/kenshoo/react-tree/blob/master
  yarn add @kenshooui/react-tree
```
Comment (liorheber) [inline packages/core/README.md:24]: lets ensure the readme files contain a link to the demo page.

File: .travis.yml [hunk 2]
```diff
@@ -40,4 +40,11 @@ deploy:
     script: if [ "$GH_TOKEN" != "false" ]; then npx auto shipit; fi;
     skip-cleanup: true
     on:
-      all_branches: true
\ No newline at end of file
+      all_branches: true
+  - provider: pages
+    local-dir: "./storybook-static"
```
Comment (liorheber) [inline .travis.yml:45]: Are you sure storybook is generated in the root folder and not in the doc folder? I'm not sure.

File: packages/core/README.md (hunk 1 above)
Comment (liatgo) [reply packages/core/README.md:24]: It will be possible only after this PR will be merged ,no?

File: .travis.yml (hunk 2 above)
Comment (liatgo) [reply .travis.yml:45]: I'm not really sure what "./storybook-static" means..I didn't find storybook-static in react-multi-select
In react tree the storybook is created inside doc folder, how can we know where is the storybook generated?

**Pull request merged**